packages/discovery-react-components/src/components/CIDocument/components/CIDocument/CIDocument.tsx

```diff
@@ -350,6 +351,7 @@ const CIDocument: FC<CIDocumentProps> = ({
               onSelectionChangeContractTabs(index, setSelectedContractFilter, resetStates)
             }
             aria-label={messages.filtersTabLabel}
+            data-testid="contract-tabs"
```

Comment:
1. There is a lot of duplication in this `<Tabs>` code; could use refactoring to cut down on that (but that's a separate issue)
2. The first case is actually invoice/PO, not contract...
3. But it seems you've noticed that only one of these is on screen at same time and we should use same test ID, to keep things simpler.
4. I suggest something like `document-info-tabs`

Reply:
Good point. Went ahead and fixed those.